src/pages/index.js

```diff
@@ -0,0 +1,337 @@
+import "./index.css";
+import Api from "../utils/api.js";
+import { enableValidation,settings,disableBtn,resetValidation} from "../scripts/validation.js";
+import {setButtonText} from "../utils/helpers.js";
+import avatarSrc from "../images/avatar.jpg";
+
+const avatarImg = document.getElementById("bessieAvatar");
+avatarImg.src = avatarSrc;
+
+import crossLogoSrc from "../images/Logo.svg";
+
+const crossLogoImg = document.getElementById("cross-logo");
+crossLogoImg.src = crossLogoSrc;
+
+
+import editPencilSrc from "../images/Pencil.svg";
+
+const pencilImg = document.getElementById("edit-pencil");
+pencilImg.src = editPencilSrc;
+
+
+import addCrossSrc from "../images/cross.svg";
+
+const addCrossImg = document.getElementById("add-cross");
+addCrossImg.src = addCrossSrc;
+
+import avatarEditPencil from "../images/avatar-edit-pencil.svg";
+
+const addAvatarEditPencil = document.getElementById("avatar-edit-pencil");
+addAvatarEditPencil.src = avatarEditPencil;
+
+
+// const initialCards = [
+//   {
+//     name: "Val Thorens",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/1-photo-by-moritz-feldmann-from-pexels.jpg",
+//   },
+//   {
+//     name: "Restaurant terrace",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/2-photo-by-ceiline-from-pexels.jpg",
+//   },
+//   {
+//     name: "An outdoor cafe",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/3-photo-by-tubanur-dogan-from-pexels.jpg",
+//   },
+//   {
+//     name: "A very long bridge, over the forest and through the trees",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/4-photo-by-maurice-laschet-from-pexels.jpg",
+//   },
+//   {
+//     name: "Tunnel with morning light",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/5-photo-by-van-anh-nguyen-from-pexels.jpg",
+//   },
+//   {
+//     name: "Mountain house",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg",
+//   },
+//   {
+//     name: "Golden Gate Bridge",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/7-photo-by-griffin-wooldridge-from-pexels.jpg",
+//   },
+// ];
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "438c56b7-9f70-465f-8796-4ec73adf5bfe",
+    "Content-Type": "application/json",
+  },
+});
+
+api.getAppInfo()
+.then(([cards, users]) => {
+  cards.forEach((card) => {
+    const cardElement = getCardElement(card);
+    cardsList.prepend(cardElement);
+  })
+profileDescription.textContent = users.about,
+profileTitle.textContent = users.name,
+avatarImg.src= users.avatar;
+})
+.catch(console.error);
```

Comment:
You need to **format** the markup (and code) to show **indentation** and to make the code more readable. You can do that with `Prettier` or a combination: press `SHIFT + ALT + F`


```js
api
  .getAppInfo()
  .then(([cards, users]) => {
    cards.forEach((card) => {
      const cardElement = getCardElement(card);
      cardsList.prepend(cardElement);
    });
    (profileDescription.textContent = users.about),
      (profileTitle.textContent = users.name),
      (avatarImg.src = users.avatar);
  })
  .catch(console.error);

```